src/console_cmds.cpp

```diff
@@ -919,6 +919,12 @@ DEF_CONSOLE_CMD(ConNetworkConnect)
 	IConsolePrintF(CC_DEFAULT, "Connecting to %s...", ip);
 	if (company != nullptr) {
 		join_as = (CompanyID)atoi(company);
+		IConsolePrintF(CC_DEFAULT, "Connecting as company id %s ...", company);
+		/* Check if we have a valid company id! */
+		if(!Company::IsValidID(join_as) && join_as != COMPANY_SPECTATOR){
```

Comment:
Needs some extra spaces
```suggestion
		if (!Company::IsValidID(join_as) && join_as != COMPANY_SPECTATOR) {
```